superset-frontend/packages/superset-ui-chart-controls/src/components/labelUtils.tsx

```diff
@@ -60,22 +96,27 @@ export const getMetricTooltipNode = (
   labelRef?: React.RefObject<any>,
 ): ReactNode => {
   // don't show tooltip if it hasn't verbose_name, label and hasn't truncated
```

Comment:
update comment: // don't show tooltip if it hasn't verbose_name, label, description, and hasn't truncated